src/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs

```diff
@@ -25,11 +25,13 @@ public CorsMiddlewareFunctionalTests(ITestOutputHelper output)
 
         public ITestOutputHelper Output { get; }
 
-        [Fact]
-        public async Task RunClientTests()
+        [Theory]
+        [InlineData("Startup")]
+        [InlineData("StartupWithoutEndpointRouting")]
+        public async Task RunClientTests(string startup)
```

Comment:
Could we make these two separate tests? `RunClientTests_WithEndpoint` and `RunClientTests_WithoutEndpoint`? Theories can be annoying to debug.